unstructured/file_utils/filetype.py

```diff
@@ -466,14 +466,20 @@ def document_to_element_list(
     num_pages = len(document.pages)
     for i, page in enumerate(document.pages):
         page_elements: List[Element] = []
+
+        page_image_metadata = _get_page_image_metadata(page)
+        image_format = page_image_metadata.get("format")
```

Comment:
Wondering if this ever returns None (because the keys always exist)

Reply:
The `image_format` would be `None` when you run this test - https://github.com/Unstructured-IO/unstructured/blob/fix/521-pdf2image-memory-error-hi-res/test_unstructured/file_utils/test_filetype.py#L475